server/ServerApp.cpp

```diff
@@ -3641,11 +3651,42 @@ void ServerApp::HandleDiplomaticStatusChange(int empire1_id, int empire2_id) {
     DiplomaticStatus status = Empires().GetDiplomaticStatus(empire1_id, empire2_id);
     DiplomaticStatusUpdateInfo update(empire1_id, empire2_id, status);
 
+    boost::posix_time::ptime timestamp = boost::posix_time::second_clock::universal_time();
+    std::string text;
+
+    const Empire* empire1 = GetEmpire(empire1_id);
+    const Empire* empire2 = GetEmpire(empire2_id);
+
+    std::string empire1_clr_str = ColorToRgbTag(empire1->Color());
+    std::string empire2_clr_str = ColorToRgbTag(empire2->Color());
+    std::string white_clr_str = "<rgba 255 255 255 255>";
```

Comment:
close the tags rather than opening more.

the default text colour is a UI setting, and cannot be assumed to be solid white anyway.